scrapy/settings/default_settings.py

```diff
@@ -91,7 +91,7 @@
 DOWNLOADER_MIDDLEWARES_BASE = {
     # Engine side
     'scrapy.downloadermiddlewares.robotstxt.RobotsTxtMiddleware': 100,
-    'scrapy.downloadermiddlewares.httpauth.HttpAuthMiddleware': 300,
+    'scrapy.downloadermiddlewares.auth.AuthMiddleware': 300,
```

Comment:
https://github.com/scrapy/scrapy/pull/1466#r38496322 made me not make this change